recipes/variant-effect-predictor/build.sh

```diff
@@ -29,3 +29,4 @@ rm -rf t/
 wget --no-check-certificate https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/master/CADD.pm
 wget --no-check-certificate https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/master/dbNSFP.pm
 wget --no-check-certificate https://raw.githubusercontent.com/konradjk/loftee/master/LoF.pm
```

Comment:
You could use such a link isn't it: https://raw.githubusercontent.com/Ensembl/VEP_plugins/435d16729a518a83b83787985987f8f2c2782e2c/dbNSFP.pm